proto/controlclient.proto

```diff
@@ -37,3 +39,19 @@ message _ListCachesResponse {
   repeated _Cache cache = 1;
   string next_token = 2;
 }
+
+message _CreateSigningKeyRequest {
+}
+
+message _CreateSigningKeyResponse {
```

Comment:
What about expiry? Should we start out with an expiration time for these secrets? E.g., 30 days to start with? Maybe we should communicate that back to the user.

Reply:
Yeah we could consider doing that too.  For now we kind of settled on having them do the key management for the private key (storing them securely, rotation, etc.).  Down the road we can add a layer of expiry on our end as well, or even change the way how key is distributed/rotated (along with our user management story?)  
@mjindalmotiv8 @danielamiao  let me know if you have any suggestions.

Reply:
Yup - the expiry is something which I was hoping the whole user management story would guide us on so I had not included it in the design doc. I do not know for example if we will have an API which they can use to see when their private key will expire. What is the behavior of the pre-signed URLs using the private key - will they stop working? Will that cause a data plane impact?

Reply:
this is not a "down the road" thing - if we ever let users generate eternal signing keys then we commit to storing their public keys forever. It's more permanent even than our current eternal jwt story...

@mjindalmotiv8 If a request uses a token that is signed by an expired signature then that request will fail as not-authorized.

I think we should start out with an expiry on these tokens, I do not think that is a reasonable fast-follow.